techniques/flash/FLASH11.html

```diff
@@ -11,14 +11,14 @@ <h3>Making a hidden description visible on request</h3>
          
             <p>In this example, an image containing statistical data is shown. The   image is provided a short textual alternative ("Graph of percentage of   total U.S. noninsitutionalized population age 16-64 declaring one or   more disabilities"). Below the image, the user can click a button that   will overlay a long textual description of the statistical information   itself. When the button is clicked, the following actions are taken: </p>
             <ul>
-               <li> The MovieClip containing the long text description is made   visible, and its <code class="prop">AccessibilityProperties.silent</code> property is   set to false to make it visible to assistive technology. Its contents   are placed in the tab order. </li>
+               <li> The MovieClip containing the long text description is made   visible, and its <code>AccessibilityProperties.silent</code> property is   set to false to make it visible to assistive technology. Its contents   are placed in the tab order. </li>
```

Comment:
```suggestion
               <li>The MovieClip containing the long text description is made visible, and its <code>AccessibilityProperties.silent</code> property is set to false to make it visible to assistive technology. Its contents are placed in the tab order. </li>
```

and it could do with highlighting